src/mono/mono/metadata/image-internals.h

```diff
@@ -9,6 +9,17 @@
 #include <mono/metadata/image.h>
 #include <mono/metadata/loader-internals.h>
 
+typedef struct {
+	int care_about_cli : 1;
+	int care_about_pecoff : 1;
+} ImageLoadOptions;
```

Comment:
nit: if it's in a header it should have a Mono prefix - `MonoImageLoadOptions`. (same with `MonoImageOpenOptions`)